File: translations/codenet/uncommented/C++2P/codellama/Code/s575600334.py
#include <iostream>
#include <cstdio>

using namespace std;

def binPrint(x, bits):
  for i in range(bits-1, -1, -1):
    print((x&(1<<i))>>i, end='')
  print()

a, b = map(int, input().split())

binPrint(a&b, 32)
binPrint(a|b, 32)
binPrint(a^b, 32)

# 

File: translations/codenet/uncommented/C++2Go/deepseek/Code/s575600334.go
package main

import (
	"fmt"
)

func binPrint(x int, bits int) {
	for i := bits - 1; i >= 0; i-- {
		fmt.Print(x & (1 << i) >> i)
	}
	fmt.Println()
}

func main() {
	var a, b int
	fmt.Scan(&a, &b)

	binPrint(a&b, 32)
	binPrint(a|b, 32)
	binPrint(a^b, 32)
}

